frontend/scss/components/molecules/code-snippet.scss

```diff
@@ -48,7 +48,7 @@ Styles taken and combined from: https://raw.githubusercontent.com/richleland/pyg
 
   pre, code {
     @include txt-mono;
-    white-space: pre-wrap;
+    white-space: pre;
```

Comment:
I thought that not wrapping causes the overflow?

Reply:
I came up with another way by adding a ```min-width: 0;``` rule on children of flex containers. It makes the code blocks look better and not wrap in weird places.

Reply:
Nice!